### PR TITLE
Work on editCourse

### PR DESCRIPTION
**Pull request by Rorr4**
Title.

### CODE REVIEW

File: scheduler/views.py [hunk 1]
```diff
@@ -43,10 +43,20 @@ def courseManagement(request):
                 return render(request, 'courseManagement.html', {'courses': courses, 'TAs': TAs, 'instructors': instructors, 'labs': labs, 'messages': "Course successfully created"})
             except ValueError as msg:
                 return render(request, 'courseManagement.html', {'courses': courses, 'TAs': TAs, 'instructors': instructors, 'labs': labs, 'messages': msg})
+        if request.method == 'POST' and 'editCourseBtn' in request.POST:
```
Comment (Kretsch-Matt) [inline scheduler/views.py:46]: It doesn't look like you have the latest views, and I think this is going to be a conflict. Can you pull and merge the latest code review into your branch? So then there wont be problems

**Review by Kretsch-Matt — CHANGES_REQUESTED** [review]

File: scheduler/views.py (hunk 1 above)
Comment (Kretsch-Matt) [reply scheduler/views.py:46]: oh youre merging to master not code review, thats why its not showing there's conflicts

Comment (Kretsch-Matt) [reply scheduler/views.py:46]: i changed the pull request to code-review instead of master

**Review by Kretsch-Matt — APPROVED** [review]

**Review by AndrewAlien — APPROVED** [review]

**Review by NathanEdwards2023 — APPROVED** [review]
Looks good.

**Pull request merged**